demos/vismotion/vismotion_demo.sh

```diff
@@ -37,20 +37,20 @@ datalad install -d . \
 
 datalad save -m 'set up done'
 
-datlad create -d . outputs/derivatives/cpp_spm-preproc
-datlad create -d . outputs/derivatives/cpp_spm-stats
+datalad create -d . outputs/derivatives/cpp_spm-preproc
+datalad create -d . outputs/derivatives/cpp_spm-stats
 
 cd code/CPP_SPM/demos/vismotion
 
 # FIX ME
-/usr/local/MATLAB/R2017a/bin/matlab \
+/Applications/MATLAB_R2017b.app/bin/matlab \
```

Comment:
hey how about we keep a commented copy of yours and mine above to show what the typical path to the matlab executables look like on Linux and Mac?

Reply:
great I dea, I actually forgot to go back yo yours path but accidentally proposed some improvements here ;)